### DISCUSSION TITLE
How to add a translation option to a module later on

### DISCUSSION THREAD

**Discussion by undersound** [area17/twill#2193] [Q&A] [answered]
What would be best practice to add the translatable option to an already created module?

Would be it for instance possible to rerun with only the -T option like `twill:make:module -T pages`

**Comment by ifox** [comment, marked as answer]
Hi @undersound, it is currently not possible to use the command to add an option to an already generated module. However adding those features is usually a matter of adding tbe corresponding trait to the model and repository class. For translations you also need to create the translation model and table. A simple approach to find out what's needed without going through all the docs on the matter is to generate another dummy module with the option you're looking for.

**Reply by undersound** [reply]
Thanks @ifox , just checking if I wasn't missing something. Dummy module works fine for now.
